app/assets/javascripts/hyrax.js

```diff
@@ -37,11 +37,16 @@
 //= require webcomponentsjs/0.5.4/CustomElements.min
 //= require time-elements
 
+//= require action_cable
+
 //= require hyrax/monkey_patch_turbolinks
+//= require hyrax/fileupload
+// Provide AMD module support
+//= require almond
```

Comment:
Moved `almond` up because it's needed by `hyrax/notification`, which needs to be loaded before `hyrax/app` is.